src/Console/Command/FixCommand.php

```diff
@@ -292,8 +294,8 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $stdErr = ($output instanceof ConsoleOutputInterface) ? $output->getErrorOutput() : null;
-        if ($stdErr && extension_loaded('xdebug')) {
+        $stdErr = ($output instanceof ConsoleOutputInterface) ? $output->getErrorOutput() : $output;
+        if (extension_loaded('xdebug')) {
```

Comment:
on txt format it could print more output (as we already do), like info about xdebug or loaded config file